src/mscorlib/src/System/Array.cs

```diff
@@ -26,14 +26,8 @@ namespace System {
     // IList<U> and IReadOnlyList<U>, where T : U dynamically.  See the SZArrayHelper class for details.
     [Serializable]
     [ComVisible(true)]
-    #if CONTRACTS_FULL
-    [ContractClass(typeof(ArrayContracts))]
-    #endif
     public abstract class Array : ICloneable, IList, IStructuralComparable, IStructuralEquatable 
     {
-        /// <internalonly/>
-        internal Array() {}  // Don't call this.  Internal for ArrayContracts type.
```

Comment:
Whoops, I think you're right. However, for any new APIs to be exposed from mscorlib they have to be exposed from [this file](https://github.com/dotnet/coreclr/blob/master/src/mscorlib/model.xml), otherwise they're automatically stripped out of the assembly by the build tools. So either way I don't think it makes a difference, since it's not listed there/isn't exposed in the contracts.